lisa/operating_system.py

```diff
@@ -323,7 +323,15 @@ def get_kernel_information(self, force_run: bool = False) -> KernelInformation:
 
     def install_packages(
         self,
-        packages: Union[str, Tool, Type[Tool], List[Union[str, Tool, Type[Tool]]]],
+        packages: Union[
+            str,
+            Tool,
+            Type[Tool],
+            List[str],
```

Comment:
Is `List` the correct type here? Typically `List` would only be used if a functionality specific to `list` is required. If the only requirement is to be an argument to `str.join()`, then it would be `Iterable`. That said, I'd probably go with `Sequence` to provide more flexibility is the subclassed methods.

Reply:
updated